addons/hr_evaluation/models/hr_evaluation.py

```diff
@@ -0,0 +1,402 @@
+            'default_partner_ids': partner_ids
+        }
+        meetings = self.env['calendar.event'].search([('partner_ids', 'in', partner_ids)])
+        res['domain'] = str([('id', 'in', meetings.ids)])
+        return res
+
+    @api.one
+    def log_meeting(self, meeting_subject, meeting_date):
+        message_receiver = []
+        if self.appraisal_manager and self.appraisal_manager_ids:
+            message_receiver.append({'survey_id': self.appraisal_manager_survey_id, 'employee_ids': self.appraisal_manager_ids})
+        if self.appraisal_self and self.appraisal_employee:
+            message_receiver.append({'survey_id': self.appraisal_self_survey_id, 'employee_ids': self.employee_id})
+        partner_ids = [emp.user_id.partner_id.id for record in message_receiver for emp in record['employee_ids'] if emp.user_id]
+        message = _("Subject: %s <br> Meeting scheduled at '%s'<br>") % (meeting_subject, meeting_date.split(' ')[0])
+        return self.message_post(body=message, partner_ids=partner_ids)
+
+
+class hr_employee(models.Model):
+    _inherit = "hr.employee"
+
+    @api.one
+    def _appraisal_count(self):
+        evaluation = self.env['hr_evaluation.evaluation']
+        self.appraisal_count = evaluation.search_count([('employee_id', '=', self.id)])
+
+    evaluation_date = fields.Date(string='Next Appraisal Date', help="The date of the next appraisal is computed by the appraisal plan's dates (first appraisal + periodicity).")
+    appraisal_manager = fields.Boolean(string='Manager')
+    appraisal_manager_ids = fields.Many2many('hr.employee', 'appraisal_manager_rel', 'hr_evaluation_evaluation_id')
+    appraisal_manager_survey_id = fields.Many2one('survey.survey', string='Manager Appraisal')
+    appraisal_colleagues = fields.Boolean(string='Colleagues')
+    appraisal_colleagues_ids = fields.Many2many('hr.employee', 'appraisal_colleagues_rel', 'hr_evaluation_evaluation_id')
+    appraisal_colleagues_survey_id = fields.Many2one('survey.survey', string="Employee's Appraisal")
+    appraisal_self = fields.Boolean(string='Employee')
+    appraisal_employee = fields.Char(string='Employee Name')
+    appraisal_self_survey_id = fields.Many2one('survey.survey', string='Self Appraisal')
+    appraisal_subordinates = fields.Boolean(string='Collaborator')
+    appraisal_subordinates_ids = fields.Many2many('hr.employee', 'appraisal_subordinates_rel', 'hr_evaluation_evaluation_id')
+    appraisal_subordinates_survey_id = fields.Many2one('survey.survey', string="collaborate's Appraisal")
+    appraisal_repeat = fields.Boolean(string='Periodic Appraisal', default=False)
+    appraisal_repeat_number = fields.Integer(string='Repeat Every', default=1)
+    appraisal_repeat_delay = fields.Selection([('year', 'Year'), ('month', 'Month')], string='Repeat Every', copy=False, default='year')
+    appraisal_count = fields.Integer(compute='_appraisal_count', string='Appraisal Interviews')
+
+    @api.onchange('appraisal_manager')
```

Comment:
- not missing `parent_id` ?
- should probably get `if self.appraisal_manager` somewhere